csharp/Platform.IO/ConsoleCancellation.cs

```diff
@@ -54,6 +82,15 @@ public void Wait()
             }
         }
 
+        /// <summary>
+        /// <para>Removes the <see cref="OnCancelKeyPress"/> handler, for the <see cref="Console.CancelKeyPress"/> event and attempts to dispose the object.</para>
+        /// <para>Удаляет обработчик <see cref="OnCancelKeyPress"/> для события <see cref="Console.CancelKeyPress"/> и выполняет попытку высвободить объект.</para>
+        /// </summary>
+        /// <param name="manual"></param>
```

Comment:
This method is inherited, so you can get missing information [by searching it in our organization](https://github.com/search?l=C%23&q=org%3Alinksplatform+DisposableBase&type=Code).